build_tools/travis/after_success.sh

```diff
@@ -18,7 +18,15 @@ if [[ $BUILD_WHEEL == true && $TRAVIS_EVENT_TYPE != pull_request ]]; then
         ANACONDA_TOKEN="$SCIKIT_LEARN_STAGING_UPLOAD_TOKEN"
     fi
 
-    pip install git+https://github.com/Anaconda-Server/anaconda-client
+    MINICONDA_URL="https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-Linux-aarch64.sh"
+    wget $MINICONDA_URL -O miniconda.sh
+    MINICONDA_PATH=$HOME/miniconda
+    chmod +x miniconda.sh && ./miniconda.sh -b -p $MINICONDA_PATH
```

Comment:
Better name this `MINIFORGE_URL` and `miniforge.sh` to avoid confusion.

Reply:
I am not getting your point 😕.